lib/package/commands.coffee

```diff
@@ -22,6 +22,14 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.eval(move: true)
+      'julia-client:run-cell': =>
```

Comment:
Hydrogen has run-cell mapped to Alt-Command-Enter (OSX) and run-cell-and-move to Alt-Shift-Enter (on OSX), I believe... 

I'd guess there are probably people like me who would map run-cell to Command-Enter (the current run-block)... Matlab's defaults are: Command-Enter (run-cell on OSX) and Command-Shift-Enter (run-cell-and-move) and Control-Enter (run-cell on Windows/Linux) and Control-Shift-Enter (run-cell-and-move on Windows/Linux). 

Not sure how to handle the multiplicity of options at the moment, so being consistent with Hydrogen (and consistent with current Juno!) is probably the best short term solution.